README.rst

```diff
@@ -0,0 +1,188 @@
+lddwrap
+=======
+.. image:: https://travis-ci.com/Parquery/lddwrap.svg?branch=master
+    :target: https://travis-ci.com/Parquery/lddwrap.svg?branch=master
+    :alt: Build Status
+
+.. image:: https://coveralls.io/repos/github/Parquery/lddwrap/badge.svg?branch=master
+    :target: https://coveralls.io/github/Parquery/lddwrap?branch=master
+    :alt: Coverage
+
+.. image:: https://badges.frapsoft.com/os/mit/mit.svg?v=10
+    :target: https://github.com/ellerbrock/open-source-badge/
+    :alt: Coverage
+
+Python3 wrapper for ldd linux command
+
+Motivation:
+===========
+Deployment scripts of our system need to determine which libraries are used. lddwrap helps us to parse the output of
+the ldd utility in a gracefully
+
+Related work:
+=============
+There exist some simple lddwrap tools. They all use the ldd command without any options, return too basic results
+(`python-ldd <https://github.com/relip/python-ldd>`_) and some of them aren't even complete projects
+(`ldd.py <https://gist.github.com/masami256/1588876>`_).
+
+This library offers a more sophisticated lddwrap. It implements the most important options of the command line ldd
+and returns a well structured list with our dependency class objects which contain all information obtained by ldd.
+
+More information:
+=================
+ldd manual page can be found `here <http://man7.org/linux/man-pages/man1/ldd.1.html>`_.
+
+Usage
+=====
+
+Get all the dependencies of the ls utility.
+
+.. code-block:: python
+
+    import pathlib
```

Comment:
Please convert these examples into doctests -- that way we can be sure that the examples are automatically verified.

You can use `...` to ignore certain parts of the output and `sorted(..., key=...)` to iterate over the dependencies in a reproducible way.